src/UserService.Business/Commands/User/EditUserCommand.cs

```diff
@@ -71,37 +69,6 @@ private async Task EditUserOfficeAsync(Guid? officeId, Guid userId, List<string>
       errors.Add(officeErrorMessage);
     }
 
-    private async Task ChangeUserRoleAsync(Guid? roleId, Guid userId, List<string> errors)
```

Comment:
do you remove endpoint from appsetings.json and madels.dll?

Reply:
this request is still using in create user command